test/test_pipeline/components/preprocessing/test_scalers.py

```diff
@@ -175,7 +175,7 @@ def test_minmax_scaler(self):
 
 
 # TODO: fix in preprocessing PR
-@unittest.skip("Skipping tests as preprocessing is not finalised")
+# @unittest.skip("Skipping tests as preprocessing is not finalised")
```

Comment:
Can we remove it rather than comment out because it is not relevant anymore?

Reply:
oh yeah. I forgot to delete them, I'll rerequest review after removing these and addressing your other comments